packages/transitive-overlay/src/index.tsx

```diff
@@ -87,14 +89,49 @@ const accessLegFilter = [
 
 type Props = {
   activeLeg?: Leg;
+  colorOverride?: string;
```

Comment:
Unlike `showRouteArrows`, `colorOverride` only applies to non-transit legs only. Is that intended? If so, could you rename `colorOverride` to `accessLegColorOverride`?

Reply:
Sure, I can do that!